### ISSUE TITLE
[1.19 -> 1.17.1] Villager Trades don't display changes in price due to demand

### ISSUE THREAD

**Issue by aPersonIsHere** [Earthcomputer/multiconnect#396]
Hello. I'm using the latest 'test' 1.19 Fabric version of this mod and playing on a 1.17.1 PaperSpigot server. After having traded with the same villager many times (with the same trade), the number of item required for trade stays the same. However, more of said item is actually needed, since a scratch mark does not appear over the original number to then render the fixed price (changed from demand tag in the trade itself). 
tldr; demand tag is not displaying properly, giving a misleading requirement for trades with a higher 'demand' value.
